README.md

```diff
@@ -2,30 +2,86 @@ Core ML Community Tools
+
+
+## Neural network conversion guide
+
+There are several `converters` available to translate neural networks trained in various frameworks into the Core ML model format.
+Following formats can be converted to the Core ML `.mlmodel` format through the coremltools python package (this repo):
+
+- Caffe V1 (.prototxt, .caffemodel format)
+- Keras API (2.3) (.h5 format)
+- Tensorflow (TF 1.13+) (.pb frozen graph def format)
+- tf.keras API (only for TF 2.0, .h5 and saved-model formats)
+
+In addition, there are two more neural network converters build on top of `coremltools`:
+- [onnx-coreml](https://github.com/onnx/onnx-coreml): to convert `.onnx` model format. Several frameworks such as PyTorch, MXNet, CaffeV2 etc
+provide native export to the ONNX format. 
+- [tfcoreml](https://github.com/tf-coreml/tf-coreml): to convert Tensorflow models. For producing Core ML models targeting iOS 13 or later, 
+tfcoreml defers to the tensorflow converter implemented inside coremltools. 
+For iOS 12 or earlier, the code path is different and lives entirely in the tfcoreml repo/python package.  
+ 
+To get an overview on how to use the converters and features such as post-training quantization using coremltools, 
+please see the [neural network guide](docs/NeuralNetworkGuide.md).  
+
+
+## Core ML specification 
+
+Core ML specifiction is fully described in a set of protobuf files. 
+They are all located in the folder `mlmodel/format/`
+
+For an overview of the Core ML framework API, see [here](https://developer.apple.com/documentation/coreml)
+
+To find the list of model types supported by Core ML, see [this](https://github.com/aseemw/coremltools/blob/f95f9b230f6a1bd8b0d9ee298b78d7786e3e7cfd/mlmodel/format/Model.proto#L229) 
+portion of the model.proto
 
-Installation
-------------
+To find the list of neural network layer types supported see [this](https://github.com/aseemw/coremltools/blob/f95f9b230f6a1bd8b0d9ee298b78d7786e3e7cfd/mlmodel/format/NeuralNetwork.proto#L472) 
+portion of the NeuralNetwork.proto
+
+Auto-generated documentation for all the protobuf files can be found at this [link](https://apple.github.io/coremltools/coremlspecification/)
+
+
+## coremltools documentation and examples
+
+Auto-generated documentation for the coremltools python package can be found at this [link](https://apple.github.io/coremltools)
+
+For many of the common utility tasks (such as loading an mlmodel, calling predict etc), 
+there is a collection of code snippets in this [document](docs/APIExamples.md)
+
+A set of python notebooks demonstrate the creation of [updatable models](examples/updatable_models)
+and [neural network inference](examples/neural_network_inference).
+More details on how to get your neural network into the Core ML format see the guide linked in the next section.  
+  
+
+## Installation
 
 We recommend using virtualenv to use, install, or build coremltools. Be
 sure to install virtualenv using your system pip.
```

Comment:
I feel like this should be combined into with the release section.

Reply:
see other comments, I want to separate out a quick install/release section and a more deep dive install section for developers

Reply:
Let me know if this is still an issue.